app/src/main/java/org/dosomething/letsdothis/ui/CampaignDetailsActivity.java

```diff
@@ -134,7 +134,7 @@ public void onUserClicked(String id)
         //FIXME
         if(BuildConfig.DEBUG)
         {
-            startActivity(PublicProfileActivity.getLaunchIntent(this));
+            startActivity(PublicProfileActivity.getLaunchIntent(this ,id));
```

Comment:
"doesnt work and crashes. add fixme if the api is not done and comment out the breaking call."